README.md

```diff
@@ -1,3 +1,134 @@
+|       0.8.0      | to be added |
+
+## Keptn CloudEvents
+
+This service reacts on the following Keptn CloudEvents (see [deploy/service.yaml](deploy/service.yaml)):
+* `sh.keptn.event.test.triggered` (used to be `sh.keptn.events.deployment-finished`) -> start locust performance tests
+* `sh.keptn.event.test.finished` (used to be`sh.keptn.events.tests-finished`) -> clean up resources and print results
+
+## Installation -  Deploy in your Kubernetes cluster
+
+To deploy the current version of the *locust-service* in your Keptn Kubernetes cluster, clone the repo and apply the [`deploy/service.yaml`](deploy/service.yaml) file:
+
+```console
+kubectl apply -f deploy/service.yaml
+```
+
+This will install the `locust-service` into the `keptn` namespace, which you can verify using:
+
+```console
+kubectl -n keptn get deployment locust-service -o wide
+kubectl -n keptn get pods -l run=locust-service
+```
+
+## Usage
+
+The `locust-service` expects locust test files in the project specific Keptn repo. It expects those files to be available in the `locust` subfolder for a service in the stage you want to execute tests.
+
+Here is an example on how to upload the `locust.py` test file via the Keptn CLI to the dev stage of project sockshop for the service carts:
+
+```
+keptn add-resource --project=sockshop --stage=dev --service=carts --resource=locust.py --resourceUri=locust/locust.py
+```
+
+### Defaults
+
+If the users does not provide a specific configuration file (see next section) the `locust-service` will search for `locust/locust.py` in the specific Keptn repo and try to execute it. If the file does not exist the tests will be skipped.
+
+### Providing a config file
+
+A `locust.conf.yaml` file can be added for more complex configurations like multiple tests files for different stages. It is also possible to provide a [locust config file](https://docs.locust.io/en/stable/configuration.html#configuration-file) to further customize the tests. If no configuration file is specified the `locust-service` will proceed with the default setting. 
+
+```
+---
+spec_version: '0.1.0'
+workloads:
+  - teststrategy: functional
+    script: /locust/basic.py
+    conf: /locust/locust.conf
+  - teststrategy: performance_light
+    script: /locust/basic.py
+    conf: /locust/locust.conf
+  - teststrategy: functional-light
+    script: /locust/basic.py
+```
+
+The contents of `locust.conf.yaml` should be used as described in the following:
+- If a "script" is given, it has to be used in the execution of the locust test.
+- If "script" is not given, we can assume that the script is part of the "config" file that is referenced.
+- If a "conf" is given, it should be used in the execution of the locust test.
+- If "conf" is not given, the "script" will be executed with default setting.
+- If both "script" and "conf" are missing, the integration skips the tests and indicate this in the result that is sent back to Keptn.
+
```

Comment:
Could you please add a sentence on how you can add the conf file to the Keptn repo?
--> `keptn add-resource ...`